nowcasting_dataset/dataset/split/method.py

```diff
@@ -0,0 +1,101 @@
+from typing import List, Tuple
+
+import numpy as np
+import pandas as pd
+
+
+def split_method(
+    datetimes: pd.DatetimeIndex,
+    train_test_validation_split: Tuple[int] = (3, 1, 1),
+    method: str = "modulo",
+    freq: str = "D",
+) -> (List[pd.Timestamp], List[pd.Timestamp], List[pd.Timestamp]):
+    """
+    Split the data by the day into train, test and validation.
+
+    method: modulo
+    If the split is (3,1,1) then, taking all the days in the dataset:
+    - train data will have all days that are modulo 5 remainder 0,1 or 2,
+    i.e 1st, 2nd, 3rd, 6th, 7th, 8th, 11th .... of the whole dataset
+    - validation data will have all days that are modulo 5 remainder 3, i.e 4th, 9th, ...
+    - test data will have all days that are modulo 5 remainder 4, i.e 5th, 10th ,  ...
+
+    method: random
+    If the split is (3,1,1) then
+    - train data will have 60% of the days
```

Comment:
Might be worth replacing 'days' with 'data' on lines 25, 26, and 27, now that this lovely new function can handle multiple frequencies :) 

```suggestion
    - train data will have 60% of the data
```